plugins/modules/database/saphana/hdbsql.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2021, Rainer Leber <rainerleber@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: hdbsql
+
+short_description: Execute SQL on HANA.
+
+version_added: "1.0.0"
+
+description: This module executes SQL statements on HANA with hdbsql.
+
+options:
+    sid:
+        description: The system ID.
+        required: true
+        type: str
+    instance:
+        description: The instance number.
+        required: false
+        type: str
+    user:
+        description: A dedicated username. Defaults to SYSTEM.
+        required: false
+        type: str
+        default: SYSTEM
+    password:
+        description: The password to connect to the database.
+        required: true
+        type: str
+    autocommit:
+        description: Autocommit the statement.
+        required: false
+        type: bool
+        default: True
+    host:
+        description: The Host ip address. The port can be defined as well.
+        required: false
+        type: str
+    database:
+        description: Define the database on which to connect.
+        required: false
+        type: str
+    encrypted:
+        description: Use encrypted connection. Defaults to false.
+        required: false
+        type: bool
+        default: false
+    filepath:
+        description:
+        - SQL query in file to run. Multiple files can be passed using YAML list syntax.
+        - Must be a string or YAML list containing strings.
+        required: false
+        type: raw
```

Comment:
I would really avoid `raw` if possible. Better use `type: list` with `elements: str`. This also allows the user to specify a single string (it will be split by commas).